projects/element-ng/landing-page/login-basic/si-login-basic.component.scss

```diff
@@ -0,0 +1,53 @@
+@use 'sass:map';
+
+@use '@siemens/element-theme/src/styles/variables';
+
+:host {
+  --animation-duration: 300ms;
+  --animation-delay: calc(var(--animation-duration) / 2);
```

Comment:
There is variables.$element-default-transition-duration.

In general also try using scss vars here. Those are checked by the compiler